src/u_websocket.c

```diff
@@ -1150,6 +1150,20 @@ int ulfius_websocket_send_message(struct _websocket_manager * websocket_manager,
   return ulfius_websocket_send_fragmented_message(websocket_manager, opcode, data_len, data, data_len);
 }
 
+/**
+ * Send a JSON message in the websocket
+ * Return U_OK on success
+ */
+#ifndef U_DISABLE_JANSSON
+int ulfius_websocket_send_json_message(struct _websocket_manager * websocket_manager,
+                                       const json_t *message) {
+  char * json = json_dumps(message, JSON_COMPACT);
+  int ret = ulfius_websocket_send_message(websocket_manager, U_WEBSOCKET_OPCODE_TEXT, strlen(json), json);
+  free(json);
```

Comment:
Stricto senso this is correct, but in ulfius' context it's better to use `o_malloc`/`o_free`, for when a program uses a different memory allocation system.

Reply:
Fixed.

Reply:
Right, I missed the call to `json_set_alloc_funcs()`.